Cargo.lock

```diff
@@ -2328,8 +2234,8 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "3a1387e07917c711fb4ee4f48ea0adb04a3c9739e53ef85bf43ae1edc2937a8b"
 dependencies = [
  "futures-io",
- "rustls",
- "webpki",
+ "rustls 0.19.1",
+ "webpki 0.21.4",
```

Comment:
it's just because we support both `native certs` and `webpki` currently in clients and we didn't want to have mutual exclusive features for it but fair enough valid concern.